### PR TITLE
Add content descriptions to playback controls

### PR DESCRIPTION
**Pull request by dturner**
I have reused the descriptions from `strings.xml`, however, there are 2 descriptions which need creating for "repeat" and "shuffle". If someone could let me know how to add these new strings, along with their translations I will add them as well.

### CODE REVIEW

File: library/ui/src/main/res/layout/exo_playback_control_view.xml [hunk 1]
```diff
@@ -31,30 +31,39 @@
     android:orientation="horizontal">
 
     <ImageButton android:id="@id/exo_prev"
+      android:contentDescription="@string/exo_controls_previous_description"
```
Comment (ojw28) [inline library/ui/src/main/res/layout/exo_playback_control_view.xml:34]: These are already added via the styles.

**Comment by ojw28** [comment]
See inline comment. The repeat and shuffle ones are set in code because the description needs to be different based on the current button state (there are already strings - `exo_controls_repeat_*_description` and `exo_controls_shuffle_*_description`).

File: library/ui/src/main/res/layout/exo_playback_control_view.xml (hunk 1 above)
Comment (dturner) [reply library/ui/src/main/res/layout/exo_playback_control_view.xml:34]: Right. Good to know - thanks. This came from Android Studio's linter....I will file a bug. 

image

**Comment by dturner** [comment]
Thanks for the review. Sorry for the noise. Android Studio bug filed: 

https://b.corp.google.com/issues/142636050

**Pull request closed without merging**